src/DialogManager.php

```diff
@@ -40,6 +40,17 @@ public function activate(Dialog $dialog): void
         $this->storeDialogState($dialog);
     }
 
+    /**
+     * @api Deactivate existing Dialog.
+     */
+    public function deactivate(Update $update): void
```

Comment:
on second thought I compared signatures of these methods:
```php
public function activate(Dialog $dialog): void;
public function deactivate(Update $update): void;
```

`activate` is straightforward, you explicitly says which dialog to activate. 

deactivate in it's current implementation forgets the active dialog state. These 2 are different conceptions: open dialog and active dialog.

 - **open**: any dialog that started but not finished yet
 - **active**: current open dialog

The packages doesn't have of limitations of a number of open dialogs.

I think such method implementation mixes these conceptions and that will lead to confusion and even bugs